applications/nrf5340_audio/src/audio/sw_codec_select.c

```diff
@@ -51,7 +58,7 @@ int sw_codec_encode(void *pcm_data, size_t pcm_size, uint8_t **encoded_data, siz
 		}
 
 		switch (m_config.encoder.num_ch) {
```

Comment:
```suggestion
		switch (m_config.encoder.audio_mode) {
```
I think this solution works best if we do it for both gateway and headset, and encode and decode